third-party/spamassassin/src/test/java/org/apache/james/spamassassin/SpamAssassinExtension.java

```diff
@@ -51,7 +52,9 @@ public SpamAssassinExtension() {
                 .withFileFromClasspath("run.sh", "docker/spamassassin/run.sh")
                 .withFileFromClasspath("spamd.sh", "docker/spamassassin/spamd.sh")
                 .withFileFromClasspath("rule-update.sh", "docker/spamassassin/rule-update.sh")
-                .withFileFromClasspath("bayes_pg.sql", "docker/spamassassin/bayes_pg.sql"));
+                .withFileFromClasspath("bayes_pg.sql", "docker/spamassassin/bayes_pg.sql"))
+                .withCreateContainerCmdModifier(cmd -> cmd.getHostConfig()
+                    .withTmpFs(ImmutableMap.of("/var/lib/postgresql/data", "rw,noexec,nosuid,size=100m")));
```

Comment:
`postgresql`? is that legal?

Reply:
Sorry? The base image is a postgres SQL image.